src/sentry/conf/server.py

```diff
@@ -1597,6 +1597,8 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:integrations-opsgenie": True,
     # Enable one-click migration from Opsgenie plugin
     "organizations:integrations-opsgenie-migration": False,
+    # Enable stacktrace linking
+    "organizations:integrations-stacktrace-link": True,
```

Comment:
is this a precursor to removing the flag?

Reply:
@ceorourke No, the flag stays. We don't read it except in the integration features UI to show it's available.

Reply:
:( feels like there should be a better way to remove flags for features that are ... not feature gated for anyone

Reply:
@ceorourke yea probably someone on the future Ecosystem team can think about how to handle these cases better